tracecontext.go

```diff
@@ -138,6 +138,12 @@ func (id SpanID) MarshalText() ([]byte, error) {
 	return text, nil
 }
 
+// SpanLink describes a linked span.
+type SpanLink struct {
+	TraceID TraceID
+	SpanID  SpanID
```

Comment:
Perhaps for consistency with the TraceContext type's field names, call these `Trace` and `Span`?